app/services/throttler/throttle_types.rb

```diff
@@ -0,0 +1,5 @@
+module Throttler
+  class ThrottleTypes
+    IDV_ACUANT = 1
```

Comment:
Since this is a model, you could move this to the model and use [active record's enum tooling](https://api.rubyonrails.org/v5.2/classes/ActiveRecord/Enum.html) there.